src/shared/widgets/image-button.jsx

```diff
@@ -0,0 +1,53 @@
+/*
+Copyright 2016 Mozilla
+
+Licensed under the Apache License, Version 2.0 (the "License"); you may not use
+this file except in compliance with the License. You may obtain a copy of the
+License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed
+under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+CONDITIONS OF ANY KIND, either express or implied. See the License for the
+specific language governing permissions and limitations under the License.
+*/
+
+import React, { PropTypes } from 'react';
+import CSSModules from 'react-css-modules';
+
+import WidgetComponent from './helpers/widget-component';
+import Styles from './image-button.css';
+import Button from './button';
+import FittedImage from './fitted-image';
+
+@CSSModules(Styles, {
+  allowMultiple: true,
+})
+export default class ImageButton extends WidgetComponent {
+  render() {
+    return (
+      <Button
+        title={this.props.title}
+        className={`${this.props.className} ${Styles.button}`}
```

Comment:
I don't see this documented anywhere in the CSS modules readmes, so let's just stick to the documentation. It might very well be a feature surfaced by webpack css modules, but we're not relying directly on their API anywhere and I don't see any reason to start here. Either access mangled style names via `this.props.styles` or simply use both `className` and `styleName` like in the rest of the codebase. You can freely mix them and all resulting class names are concatenated.